docs/spec_draft.md

```diff
@@ -343,6 +344,78 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_grad
+
+### Semantics
+
+Computes gradients of `Batch Normalizing Transform` for `operand`, `scale` and
+offset, for each feature in the `feature_index` dimension, using `grad_output`
```

Comment:
Is there a specific reason we mention `grad_output` but not the other operands.

Reply:
`grad_output` is the gradient that would be "back-propagated", which is why I mentioned it separately.